metadata-ingestion/src/datahub/emitter/rest_emitter.py

```diff
@@ -63,6 +64,10 @@ class DatahubRestEmitter:
     _session: requests.Session
 
     def __init__(self, gms_server: str, token: Optional[str] = None):
+        if ":9002" in gms_server:
```

Comment:
Can this be better detected by hitting some diagnostic endpoint where the server actually responds with "what" it is?

Reply:
Yep - added that to my TODO list